modules/ROOT/pages/syntax/parsing.adoc

```diff
@@ -0,0 +1,41 @@
+:description: This section describes whitespace in Cypher
+
+[[parsing]]
+= Parsing
+
+[abstract]
+--
+This section provides a general overview of how Cypher is parsing an input String, including whitespace.
+--
+
+Cypher considers input Strings as a sequence of Unicode UTF-16 code points. Unicodes can generally be escaped as `\uxxx`.
+
+Additional documentation on escaping rules for string literals, names and regular expressions can be found here:
+
+* xref::syntax/expressions.adoc#cypher-expressions-string-literals[String literal escape sequences]
+* xref::syntax/naming.adoc#symbolic-names-escaping-rules[Using special characters in names]
+* xref::clauses/where.adoc#escaping-in-regular-expressions[Regular epxressions]
+
+[[cypher-whitespaces]]
+== Supported Whitespace
+
+In Cypher, whitespace can be used as a separator between keywords but has no function otherwise.
+
+The following characters are considered as whitespace:
+
+[options="header", cols=">1,<2"]
+|===
+| Description                       | List of included Unicode Characters
+| Unicode General Category Zp       | \u2029
```

Comment:
The rest of the tables, like https://neo4j.com/docs/cypher-manual/current/syntax/expressions/#cypher-expressions-string-literals, use monospaced font, i.e. use `u2029` instead of \u2029 please